packages/nextjs/app/page.tsx

```diff
@@ -19,6 +20,77 @@ const Home: NextPage = () => {
     }
   }
 
+  const { address } = useAccount();
+
+  const { data: Ens } = useEnsName({
+    address,
+    chainId: 1,
+  });
+
+  const { data: allowStatus } = useScaffoldContractRead({
+    contractName: "BatchRegistry",
+    functionName: "allowList",
+    args: [address],
+  });
+
+  const { data: checkedInStatus } = useScaffoldContractRead({
+    contractName: "BatchRegistry",
+    functionName: "yourContractAddress",
+    args: [address],
+  });
+  const CheckStatus = useCallback(() => {
+    if (!address) return <span>Please connect your wallet to see feature more details.</span>;
+
+    const taskUrl = "https://github.com/BuidlGuidl/batch3.buidlguidl.com/issues";
+    const issueUrl = taskUrl + "/9";
```

Comment:
nitpick: I think we can hardcode the url directly instead appending "/9" to it, we don't need an extra variable